lib/knackhq/client.rb

```diff
@@ -0,0 +1,42 @@
+require 'knackhq/client/version'
+require 'blanket'
+require 'hashie'
+
+module Knackhq
+  class Client
+    attr_accessor :base_uri, :x_knack_application_id, :x_knack_rest_api_key
+
+    def initialize(base_uri, x_knack_application_id, x_knack_rest_api_key)
+      @base_uri = base_uri
+      @x_knack_application_id = x_knack_application_id
+      @x_knack_rest_api_key = x_knack_rest_api_key
+    end
+
+    def objects
```

Comment:
Part of me says that it would be nice to verify that the request is valid for the Knack API (ie, objects, object_*), that being said if Blanket already handles invalid requests then we can certainly use that initially and build on if necessary.